mbgl/camera_options.go

```diff
@@ -1,9 +1,10 @@
 package mbgl
-import "C"
 
 /*
+#cgo CXXFLAGS:-I${SRCDIR}/../mapbox-gl-native/include
+#cgo CXXFLAGS:-I${SRCDIR}/../mapbox-gl-native/platform/default
```

Comment:
Could you move these to `mbgl.go`, I want to keep the compiler directives in one file to easily keep track of them.

Reply:
Sure, we will need to have these things be on a per OS bases soon. I did this just to get this out.

Reply:
It is currently split up by OS (see `mbgl_darwin.go` and `mbgl_linux.go`) although, I haven't tested the bindings on linux.